src/main/java/xyz/thepathfinder/android/TransportListener.java

```diff
@@ -1,5 +1,47 @@
 package xyz.thepathfinder.android;
 
-public abstract class TransportListener implements Listener {
+import com.google.gson.JsonObject;
 
+/**
+ * The <tt>TransportListener</tt> interface gives developers access to notifications
+ * when a transport is modified. The notifications also include route notifications.
+ *
+ * @author David Robinson
+ */
+public interface TransportListener extends Listener<Transport> {
+
+    /**
+     * Invoked when the transport receives a routed message.
+     *
+     * @param route the transport's new route.
+     */
+    public void routed(Route route);
+
+    /**
+     * Invoked when the transport's latitude was updated by a message.
+     *
+     * @param latitude that the transport updated to.
+     */
+    public void latitudeUpdated(double latitude);
```

Comment:
I expose a method to update both at the same time. Someone might want to use it.